### ISSUE TITLE
Synchronizing the scale widget with the grid size

### ISSUE THREAD

**Issue by BlueSteelAUS** [toolbox4minecraft/amidst#111]
when i zoom in / out using mouse its saying 500 blocks at the last 1-2-3 1024 blocks actually being displayed when zooming in

**Comment by stefandollase** [comment]
I just opened up Amidst to try this, however I am not able to reproduce the issue. I do not even fully understand what you mean. Can you clarify this? A screenshot would be nice to see what is going on.

**Comment by BlueSteelAUS** [comment]
ok. i scroll out till the grid is 1024 , then i start scrolling back in, the bar ar the bottom middle of the screen (scale) drops down to 512 before the grid does, by 3 zooms, then the grid again matches the scale of 512

**Comment by stefandollase** [comment]
Thanks for the clarification. The number of blocks displayed in the scale widget matches the length of the line in the scale widget. It is not supposed to match the grid size.

If I understand you correctly, you are not saying that the length of the line in the scale widget is wrong. You are saying that the number of blocks in the scale widget does not match the current grid size. Did I get this right?

**Comment by BlueSteelAUS** [comment]
yes. its still up stating 512 when its still around 1024. it only truely gets to 512 when the grid changes size

**Comment by stefandollase** [comment]
In this case, it is indeed working as intended, since the grid is not supposed to be synchronized with the scale widget. However, when there is time, we might look into synchronizing them. Thus, this is now a feature request.

**Comment by BlueSteelAUS** [comment]
ty

**Comment by BigAlanM** [comment]
Synchronize it, how?
As it is now, as far as I can tell, it stays between a minimum pixel length and a maximum pixel length which is a pretty good approximation of how the map scale is displayed on a paper map.
The difference being paper map scale stays the same size and just changes the values, which means that the map scaling has to be done in certain fixed magnifications.
Scaling on Amidst is a fairly smooth zoom function in Windows using the scroll wheel of the mouse and as such I like it because I can expand or contract a map so that the first ring of Strongholds are just being displayed.

**Comment by BlueSteelAUS** [comment]
ok i see whats happening now. its all good. feel  free to close this suggestion

**Comment by stefandollase** [comment]
This did indeed have a point. However, it is hard to describe. For now, I will just reopen it.

The change would only alter the behavior of the grid while scrolling. It would not alter the scrolling itself.
